util/build_yang_test.go

```diff
@@ -0,0 +1,55 @@
+package util
```

Comment:
Should we move `util` to `pkg/util` -- since this is where we have the non-binary code in `goyang`?